engines/ic3.cpp

```diff
@@ -51,7 +51,7 @@ IC3Formula IC3::get_model_ic3formula(TermVec * out_inputs,
   const UnorderedTermSet & statevars = ts_->statevars();
   TermVec children;
   children.reserve(statevars.size());
-  for (auto sv : ts_->statevars()) {
+  for (auto const &sv : ts_->statevars()) {
```

Comment:
I have a slight preference for `const auto &`. They're semantically the same but since elsewhere we write `const <type-specifier> &`, I think it's more consistent.

Reply:
done in 4a22594